src/_pytest/capture.py

```diff
@@ -351,11 +351,17 @@ def close(self):
             self._capture.stop_capturing()
             self._capture = None
 
-    def readouterr(self):
+    def readouterr(self, combined=False, flush=True):
         """Read and return the captured output so far, resetting the internal buffer.
 
         :return: captured content as a namedtuple with ``out`` and ``err`` string attributes
         """
+        if combined:
+            return CaptureResult(self._get_combined(), None)
```

Comment:
Just a quick thought: couldn't `CaptureResult` gain an optional `combined` attribute/property?
It would still provide `out` and `err´ then, but would also have `combined`.

`combined` could even default to `out + err` then (when not combining).

(I think it's bad to actually have the combined output as `out` here)

Reply:
Yeah, I was thinking about it initially, but since the `CaptureResult` is a `namedtuple` I see 2 potential solutions:  
- It should be changed to regular class with a new attribute or a method, that behaves as tuple to preserve current behavior.
- Instead of returning `CaptureResult` for `combined=True`, just return a single string with the combined output.

I think both approaches are bad - the first one has no added value and may cause confusion, and the second one will make `readouterr()` inconsistent and also potentially confusing.

Maybe adding a new `read()` or `readcombined()` method with a different return type would make sense here?